hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/spi/state/impl/StatesImpl.java

```diff
@@ -39,17 +45,21 @@ public void updateChildren(final ServicesState sourceState) {
     }
 
     @Override
-    public @Nonnull <K, V> State<K, V> get(@Nonnull final String stateKey) {
+    public @Nonnull <K, V> State<K, V> get(@Nonnull final StateKey stateKey) {
         Objects.requireNonNull(stateKey);
 
-        if (stateKey.equals(ACCOUNTS_KEY)) {
+        if (stateKey.equals(ACCOUNT_STORE)) {
             final var state =
                     new InMemoryStateImpl<>(stateKey, children.accounts(), children.signedAt());
             return (StateBase) state;
+        } else if (stateKey.equals(ALIASES_STORE)) {
+            final var state =
+                    new RebuiltStateImpl<>(stateKey, children.aliases(), children.signedAt());
+            return (StateBase) state;
+        } else {
+            throw new NotImplementedException(
+                    String.format("State key %s not supported", stateKey));
         }
-        // Will be adding other keys as needed. This is the only key needed for signature
-        // verification.
-        throw new IllegalArgumentException("State key provided is not defined");
```

Comment:
Whoops, I didn't see this line when reviewing earlier, so you'd already addressed the comment I made. Sorry about that